extensions/2.0/Vendor/ADOBE_materials_clearcoat_tint/README.md

```diff
@@ -0,0 +1,79 @@
+# ADOBE\_materials\_clearcoat\_tint
+
+## Contributors
+
+* Mike Bond, Adobe, [@miibond](https://twitter.com/miibond)
+
+## Status
+
+Complete
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+* Requires the `KHR_materials_clearcoat` extension to also be defined on the material
+
+## Exclusions
+
+* This extension must not be used on a material that also uses `KHR_materials_pbrSpecularGlossiness`.
+* This extension must not be used on a material that also uses `KHR_materials_unlit`.
+
+## Overview
+
+This extension defines a way to tint the clear coating provided by the `KHR_materials_clearcoat` extension.
+
+## Extending Materials
+
+Adding tint to clearcoat materials can be done by adding the `ADOBE_materials_clearcoat_tint` extension to any glTF material that already has the `KHR_materials_clearcoat` extension defined on it.  For example, the following defines a material with a dark red varnish.
+
+```json
+{
+    "materials": [
+        {
+            "name": "varnish",
+            "extensions": {
+                "KHR_materials_clearcoat": {
+                    "clearcoatFactor": 1.0
+                },
+                "ADOBE_materials_clearcoat_tint": {
+                    "clearcoatTintFactor": [
+                      0.4,
+                      0.1,
+                      0.1
+                    ]
+                }
+            }
+        }
+    ]
+}
+```
+
+### Clearcoat Tint
+
+All implementations should use the same calculations for the BRDF inputs. Implementations of the BRDF itself can vary based on device performance and resource constraints. See [Appendix B](/specification/2.0/README.md#appendix-b-brdf-implementation) for more details on the BRDF calculations.
+
+As with the core glTF 2.0 spec, all coloured textures are assumed to be sRGB and all coloured factors are assumed to be linear.
+
+|                                  | Type                                                                            | Description                            | Required             |
+|----------------------------------|---------------------------------------------------------------------------------|----------------------------------------|----------------------|
+|**clearcoatTintFactor**               | `number[3]`                                                                        | The clearcoat tint's colour factor.         | No, default: `[1.0, 1.0, 1.0]`   |
+|**clearcoatTintTexture**              | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo)             | The clearcoat layer's tint texture. | No                   |
```

Comment:
The link to the base spec should be updated.

Reply:
Done.